Automattic/wp-calypso
# Activity log: update banners

#### Changes proposed in this Pull Request

* Remove credentials banner
* Remove backup status banner
* Update upgrade banner
* Update intro banner
* Update intro banner associated styles

#### Testing instructions

* Fire up this PR
* Create a new site, select the free Jetpack plan, and ensure you only see one intro banner on the AL. When dismissed, you should see the permanent upgrade nudge.
* Create a new site, select any paid plan with Backup, and ensure you only see one intro banner on the AL. When dismissed, you should see no banners.

Fixes https://github.com/Automattic/wp-calypso/issues/45735
See p6TEKc-3U2-p2

#### Screenshots

Before, free plan:
![image](https://user-images.githubusercontent.com/390760/93624951-11cba680-f9d9-11ea-83f3-f479a1853037.png)

After, free plan:
![image](https://user-images.githubusercontent.com/390760/93624917-feb8d680-f9d8-11ea-812a-00959ee20ad6.png)


Before, paid plan (missing the credentials and backup status banner):
![image](https://user-images.githubusercontent.com/390760/93625058-358eec80-f9d9-11ea-9e8a-d54e5e2955db.png)



After, paid plan:
![image](https://user-images.githubusercontent.com/390760/93625098-493a5300-f9d9-11ea-99f4-0591097a19f5.png)
